src/test/resources/basicPom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am wondering whether this file should be moved to `src/test/resources/DifferentFilesTest/pom.xml` instead?

Reply:
moved/renamed